<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebPhonon-Qt5</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/Icons/WebPhononIcon.png</normaloff>:/Icons/WebPhononIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="1">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="5" column="1">
     <layout class="QGridLayout" name="gridLayout"/>
    </item>
    <item row="5" column="2">
     <widget class="Phonon::VolumeSlider" name="volumeSlider">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="Phonon::SeekSlider" name="seekSlider"/>
      </item>
      <item>
       <widget class="QLabel" name="hourlabel_1">
        <property name="text">
         <string>HH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minlabel_1">
        <property name="text">
         <string>MM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="seclabel_1">
        <property name="text">
         <string>SS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>/</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="hourlabel_2">
        <property name="text">
         <string>HH</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minlabel_2">
        <property name="text">
         <string>MM</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="seclabel_2">
        <property name="text">
         <string>SS </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Amplifier dB Level </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>834</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionLocal_File"/>
     <addaction name="actionStream"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPlayer">
    <property name="title">
     <string>Player</string>
    </property>
    <widget class="QMenu" name="menuAspect_Ratio">
     <property name="title">
      <string>Aspect Ratio</string>
     </property>
     <addaction name="actionAuto"/>
     <addaction name="actionSquare"/>
     <addaction name="actionWideScreen"/>
    </widget>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionFull_Screen"/>
    <addaction name="actionBack"/>
    <addaction name="actionFoward"/>
    <addaction name="actionShow_Playlist"/>
    <addaction name="actionVolume_Up"/>
    <addaction name="actionVolume_Down"/>
    <addaction name="actionSkip_Foward"/>
    <addaction name="actionSkip_Backward"/>
    <addaction name="actionClear_Playlist"/>
    <addaction name="menuAspect_Ratio"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPlayer"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLocal_File"/>
   <addaction name="actionStream"/>
   <addaction name="actionBack"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="actionFoward"/>
   <addaction name="actionFull_Screen"/>
   <addaction name="actionShow_Playlist"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLocal_File">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/document-open.png</normaloff>:/Icons/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Local File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionStream">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/applications-internet.png</normaloff>:/Icons/applications-internet.png</iconset>
   </property>
   <property name="text">
    <string>Stream</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/application-exit.png</normaloff>:/Icons/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/media-playback-start.png</normaloff>:/Icons/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="shortcut">
    <string>Media Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/media-playback-pause.png</normaloff>:/Icons/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/media-playback-stop.png</normaloff>:/Icons/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Media Stop</string>
   </property>
  </action>
  <action name="actionFull_Screen">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/video-television.png</normaloff>:/Icons/video-television.png</iconset>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/media-skip-backward.png</normaloff>:/Icons/media-skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionFoward">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/media-skip-forward.png</normaloff>:/Icons/media-skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>Foward</string>
   </property>
  </action>
  <action name="actionShow_Playlist">
   <property name="icon">
    <iconset resource="Icons.qrc">
     <normaloff>:/Icons/view-close.png</normaloff>:/Icons/view-close.png</iconset>
   </property>
   <property name="text">
    <string>Show Playlist</string>
   </property>
  </action>
  <action name="actionVolume_Up">
   <property name="text">
    <string>Volume Up</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionVolume_Down">
   <property name="text">
    <string>Volume Down</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionSkip_Foward">
   <property name="text">
    <string>Skip Foward</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionSkip_Backward">
   <property name="text">
    <string>Skip Backward</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionClear_Playlist">
   <property name="text">
    <string>Clear Playlist</string>
   </property>
  </action>
  <action name="actionAuto">
   <property name="text">
    <string>Auto</string>
   </property>
   <property name="shortcut">
    <string>1</string>
   </property>
  </action>
  <action name="actionSquare">
   <property name="text">
    <string>Square</string>
   </property>
   <property name="shortcut">
    <string>2</string>
   </property>
  </action>
  <action name="actionWideScreen">
   <property name="text">
    <string>Wide Screen</string>
   </property>
   <property name="shortcut">
    <string>3</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
